about_processes.exs

```diff
@@ -0,0 +1,37 @@
+defmodule AboutProcesses do
+  use Koans
+
+  think "You are a process with a process ID" do
+    pid_name = inspect(self)
+
+    assert_? pid_name =~ ~r/#PID<\d+\.\d+\.\d+>/
+  end
+
+  think "Processes receive messages" do
+    send self, {:hello, "world"}
+
+    receive do
+      {:hello, message} -> assert message == __?
+    end
+  end
+
+  think "A spawned process is a function running in parallel" do
+    spawn __?
```

Comment:
Yeah, I can't figure out an example that
1. Doesn't turn into the example below
2. Doesn't rely on capturing `stdout` and making sure that the spawned function is writing something to `stdout` (I couldn't even figure out how to do that but it would probably be very confusing)